<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSvgEditorClass</class>
 <widget class="QMainWindow" name="KSvgEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>604</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KSvgEditor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="toolBar" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="editPage" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 127);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="paramBar" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(255, 85, 127);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ksvgeditor.qrc"/>
 </resources>
 <connections/>
</ui>
